tests/conftest.py

```diff
@@ -1,393 +1,741 @@
-            }
-        )
-
-    def upload(*args, **kwargs) -> None:
-        name = ""
-        for k, v in kwargs.items():
-            if isinstance(v, Path) or (isinstance(v, str) and Path(v).exists()):
-                kwargs[k] = "/".join(Path(v).relative_to(TEST_FOLDER).parts)
-                name = Path(v).name
-
-        written_resources[resource_cls.__name__].append(
-            {
-                "name": name,
-                "args": list(args),
-                "kwargs": dict(kwargs),
-            }
-        )
-
-    def apply_dml(*args, **kwargs):
-        data = dict(kwargs)
-        data["args"] = list(args)
-        written_resources[resource_cls.__name__].append(data)
-
-    def delete_core(
-        id: int | Sequence[int] | None = None,
-        external_id: str | Sequence[str] | None = None,
-        **_,
-    ) -> list:
-        deleted = []
-        if not isinstance(id, str) and isinstance(id, Sequence):
-            deleted.extend({"id": i} for i in id)
-        elif isinstance(id, int):
-            deleted.append({"id": id})
-        if isinstance(external_id, str):
-            deleted.append({"externalId": external_id})
-        elif isinstance(external_id, Sequence):
-            deleted.extend({"externalId": i} for i in external_id)
-        if deleted:
-            deleted_resources[resource_cls.__name__].extend(deleted)
-        return deleted
-
-    def delete_data_modeling(ids: VersionedDataModelingId | Sequence[VersionedDataModelingId]) -> list:
-        deleted = []
-        if isinstance(ids, (VersionedDataModelingId, InstanceId)):
-            deleted.append(ids.dump(camel_case=True))
-        elif isinstance(ids, Sequence):
-            deleted.extend([id.dump(camel_case=True) for id in ids])
-        if deleted:
-            deleted_resources[resource_cls.__name__].extend(deleted)
-        return deleted
-
-    def delete_instances(
-        nodes: NodeId | Sequence[NodeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
-        edges: EdgeId | Sequence[EdgeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
-    ) -> list:
-        deleted = []
-        if isinstance(nodes, NodeId):
-            deleted.append(nodes.dump(camel_case=True, include_instance_type=True))
-        elif isinstance(nodes, tuple):
-            deleted.append(NodeId(*nodes).dump(camel_case=True, include_instance_type=True))
-        elif isinstance(edges, EdgeId):
-            deleted.append(edges.dump(camel_case=True, include_instance_type=True))
-        elif isinstance(edges, tuple):
-            deleted.append(EdgeId(*edges).dump(camel_case=True, include_instance_type=True))
-        elif isinstance(nodes, Sequence):
-            deleted.extend(
-                [
-                    node.dump(camel_case=True, include_instance_type=True) if isinstance(node, NodeId) else node
-                    for node in nodes
-                ]
+        def apply_dml(*args, **kwargs):
```

Comment:
Didn't we remove graphql/DML upload support?

Reply:
Yes, we did. Will remove it.